sample/kotlin/src/main/kotlin/com/datadog/android/sample/NavActivity.kt

```diff
@@ -48,18 +57,30 @@ class NavActivity : AppCompatActivity() {
             R.id.navigation_view_pager -> {
                 startActivity(Intent(this, ViewPagerActivity::class.java))
             }
-            R.id.show_snack_bar->{
+            R.id.show_snack_bar -> {
                 Snackbar.make(
                     this.window.decorView.rootView,
                     "Demo message",
                     Snackbar.LENGTH_LONG
                 ).show()
-
             }
             else -> result = super.onOptionsItemSelected(item)
         }
         return result
     }
 
+    /**
+     * A native method that is implemented by the 'native-lib' native library,
+     * which is packaged with this application. It will throw a C++ exception
+     * and catch it in the signal handler which will be visible in the logs.
+     */
+    external fun crashAndGetExceptionMessage()
+
     // endregion
+
+    companion object {
+        init {
+            System.loadLibrary("native-lib");
```

Comment:
How does the exception look like for this crash?

Reply:
There's no exception...for now we are just sending a log to the JVM log as explained above. I will get to that point soon, not sure exactly now how I can spawn an Exception from there to JVM.